cpu/atmega_common/cpu.c

```diff
@@ -110,28 +125,31 @@ void cpu_init(void)
  * EIFR – External Interrupt Flag Register
  * PCIFR – Pin Change Interrupt Flag Register
  */
-ISR(BADISR_vect){
-
+ISR(BADISR_vect)
+{
     _reset_cause();
 
-    printf_P(PSTR("FATAL ERROR: BADISR_vect called, unprocessed Interrupt.\n"
-                  "STOP Execution.\n"));
-
+#if defined (CPU_ATMEGA256RFR2)
     printf("IRQ_STATUS %#02x\nIRQ_STATUS1 %#02x\n",
-            (unsigned int)IRQ_STATUS, (unsigned int)IRQ_STATUS1 );
+            (unsigned int)IRQ_STATUS, (unsigned int)IRQ_STATUS1);
 
-    printf("SCIRQS %#02x\nBATMON %#02x\n", (unsigned int)SCIRQS, (unsigned int)BATMON );
+    printf("SCIRQS %#02x\nBATMON %#02x\n", (unsigned int)SCIRQS, (unsigned int)BATMON);
 
-    printf("EIFR %#02x\nPCIFR %#02x\n", (unsigned int)EIFR, (unsigned int)PCIFR );
+    printf("EIFR %#02x\nPCIFR %#02x\n", (unsigned int)EIFR, (unsigned int)PCIFR);
+#endif
 
-    /* White LED light is used to signal ERROR. */
-    LED_PORT |= (LED2_MASK | LED1_MASK | LED0_MASK);
+    /* Use LED light to signal ERROR. */
+    LED_PANIC;
```

Comment:
I don't think there are any now, but we might want to ```#ifdef LED_PANIC``` this for boards that don't have LEDs.

Reply:
Done.